src/component/About/About.jsx

```diff
@@ -4,8 +4,8 @@ import './About.css';
 const About = () => {
     return (
         <>
-            <div className='about-content'>
-                <div className='About-container'>
+            <div className='about-container'>
+                <div className='About-child'>
                     <div className='Aboutimg'>
```

Comment:
Your class naming should be persistent and its good to go with the popular convention which is snake case. So you may want to change the classname to snake case. Look out for everywhere in your codebase you didn't use snake case so that you'll make changes for all.